app/controllers/user.js

```diff
@@ -35,3 +35,15 @@ exports.signIn = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.getUsers = async (req, res, next) => {
+  try {
+    const { limit, page } = req.query;
+    const offset = page * limit;
```

Comment:
This should be like this, so the offset can be 0
```suggestion
    const offset = (page - 1) * limit;
```